src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -137,13 +137,13 @@ public static object GetUninitializedObjectWithFormatterServices(Type type)
         private static SerializationContext serializationContext;
 
         private static SerializationContext CurrentSerializationContext
-            => serializationContext ?? (serializationContext = new SerializationContext());
+            => serializationContext ?? (serializationContext = new SerializationContext(null));
         
         [ThreadStatic]
         private static DeserializationContext deserializationContext;
 
         private static DeserializationContext CurrentDeserializationContext
-            => deserializationContext ?? (deserializationContext = new DeserializationContext());
+            => deserializationContext ?? (deserializationContext = new DeserializationContext(null));
```

Comment:
No, this is fine - it will be set to RuntimeClient.Current upon usage if it wasn't set here (because of static type initialization order)

This is fixed in the subsequent PR for non-static serialization manager:
![image](https://cloud.githubusercontent.com/assets/203839/23045303/47683946-f4f8-11e6-8afa-d75d04eaa246.png)